src/pudl/io_managers.py

```diff
@@ -96,6 +97,10 @@ def __init__(
             db_name: the name of sqlite database.
             md: database metadata described as a SQLAlchemy MetaData object. If not specified,
                 default to metadata stored in the pudl.metadata subpackage.
+            timeout: How many seconds the connection should wait before raising
+                     an exception, if the database is locked by another connection.
+                     If another connection opens a transaction to modify the database,
+                     it will be locked until that transaction is committed.
```

Comment:
Are these indents being added by an autoformatter? We've been following the [Google docstring format](https://google.github.io/styleguide/pyguide.html#doc-function-args) which uses a 4 (or 2) space indent on lines after the argument.

Reply:
Ah sorry. Just fixed the formatting.